bin/build_containers.sh

```diff
@@ -55,25 +55,30 @@ fi
 
 for container in server celery redis websocket nginx viewer wifi-connect 'test'; do
```

Comment:
We should probably turn this list into an array so that it can be passed on from command line.

Reply:
Got it. This will be done on a separate PR.